apps/avifenc.c

```diff
@@ -242,11 +242,12 @@ static avifBool readEntireFile(const char * filename, avifRWData * raw)
     }
 
     fseek(f, 0, SEEK_END);
-    size_t fileSize = (size_t)ftell(f);
-    if (!fileSize) {
+    long pos = ftell(f);
+    if (pos <= 0) {
```

Comment:
Joe: I kept the test for equality to 0. I assume you want to treat an empty file as error.